### ISSUE TITLE
Referring other steps should be renamed

### ISSUE THREAD

**Issue by agentmilindu** [leopardslab/dunner#58]
```
deploy:
- image: 'node'
  command: ['node', '--version']
- name: '@status'
```

makes no sense, let's rename it to 
```
deploy:
- image: 'node'
  command: ['node', '--version']
- from: 'status'
```
I'm open for suggesions also.

**Comment by ayushjn20** [comment]
Working on this. 
Suggestion: Instead of using keyword as `from: 'status'`, `follow: '@status'` seems more intuitive. What do you suggest @agentmilindu ?

**Comment by agentmilindu** [comment]
`follow` seems good, shall we make it `follow: 'status'`?

**Comment by ayushjn20** [comment]
Closed with reference to #64
